tests/units/anta_tests/test_interfaces.py

```diff
@@ -1190,7 +1190,10 @@
         },
         "expected": {
             "result": "failure",
-            "messages": ["The following interface(s) are not in the expected state: ['Ethernet2 is up/unknown'"],
+            "messages": [
+                "Ethernet2 - Expected: up/down, Actual: up/unknown",
+                "Ethernet8 - Expected: up/up, Actual: up/down",
```

Comment:
should we add a test for when we expect status `down` and not protocol and actual status is `up`. Feels like it is not covered here

Reply:
Added unit test.